geniza/corpus/tests/test_ja.py

```diff
@@ -37,8 +41,42 @@ def test_arabic_or_ja__arabic():
 
 
 def test_arabic_or_ja_exact_phrase():
-    assert arabic_or_ja('"تعطل شغله"', boost=False) == '"(تعطل|תעטל) (شغله|שגלה)"'
+    # make sure basic or is working
+    assert arabic_or_ja('"تعطل شغله"', boost=False) in {
+        '("תעטל שגלה"|"تعطل شغله")',
+        '("تعطل شغله"|"תעטל שגלה")',
+    }
```

Comment:
do you still need the variations for the tests to pass locally? would like to clean these up to assert only what we expect, which should be arabic first based on the way you're constructing it

Reply:
I still need them. darned if I know why

Reply:
so weird! after you've done your other cleanup, I'll pull this branch and see if it works for me locally without them, and we can check it if works in CI too